### PR TITLE
RavenDB-5052 change from IndexHandler to StudioHandler

### PR DESCRIPTION
**Pull request by IdanHaim**

### CODE REVIEW

File: src/Raven.Server/Web/System/StudioHandler.cs [hunk 1]
```diff
@@ -120,5 +123,26 @@ public Task RavenRoot()
             HttpContext.Response.StatusCode = 301;
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/c-sharp-index-definition", "GET")]
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/StudioHandler.cs:127]: @IdanHaim The intention is to have Studio related endpoints under /studio/*\* (or sth similar), not only to move them to the StudioHandler

Comment (IdanHaim) [reply src/Raven.Server/Web/System/StudioHandler.cs:127]: Yea I thought about it, but after the databases/ you will put the database name and that how we work until now. dont care to change it but in my opinion it is Less confusing that way

Comment (ppekrol) [reply src/Raven.Server/Web/System/StudioHandler.cs:127]: I know, but now there is chaos, and this is clearly studio related endpoint.

Another option is to move this action under
/databases/*/indexes/c-sharp-index-definition
like here
https://github.com/ayende/ravendb/blob/v4.0/src/Raven.Server/Documents/Handlers/IndexHandler.cs#L54

Comment (ayende) [reply src/Raven.Server/Web/System/StudioHandler.cs:127]: Yes, let us go with this.

**Pull request closed without merging**